fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -573,7 +565,8 @@ Reference<IDatabase> DLApi::createDatabase609(const char* clusterFilePath) {
 		                                              innerApi->clusterCreateDatabase(cluster.get(), (uint8_t*)"DB", 2),
 		                                              [](FdbCApi::FDBFuture* f, FdbCApi* api) {
 			                                              FdbCApi::FDBDatabase* db;
-			                                              api->futureGetDatabase(f, &db);
+			                                              auto err = (api->futureGetDatabase(f, &db));
```

Comment:
Sorry, I forgot to remove some parentheses in my previous suggestion:

```suggestion
			                                              auto err = api->futureGetDatabase(f, &db);
```